nomad/local/integration-tests.nomad

```diff
@@ -270,40 +169,73 @@ job "integration-tests" {
         }
       }
     }
-    task "graphql-endpoint-tests" {
-      driver = "docker"
-
-      config {
-        image      = "${var.container_registry}grapl/graphql-endpoint-tests:dev"
-        entrypoint = ["/bin/bash", "-o", "errexit", "-o", "nounset", "-c"]
-        command    = "cd graphql_endpoint_tests; py.test --capture=no -n 1 -m 'integration_test'"
-      }
 
-      env {
-        # aws vars
-        AWS_REGION                  = var.aws_region
-        GRAPL_AWS_ENDPOINT          = local.aws_endpoint
-        GRAPL_AWS_ACCESS_KEY_ID     = var.aws_access_key_id
-        GRAPL_AWS_ACCESS_KEY_SECRET = var.aws_access_key_secret
+    volume "grapl-root-volume" {
+      # The definition of this `grapl-root-volume` is written as a Nomad agent
+      # config in `start_detach.sh`
+      type      = "host"
+      source    = "grapl-root-volume"
+      read_only = false
+    }
```

Comment:
^ This is where we consume the infrastructure I made in https://github.com/grapl-security/grapl/pull/1190

actual-new-change-from-this-pr